app/models/finance/statement/ecf/paid.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Finance::Statement::ECF::Paid < Finance::Statement::ECF
+  has_many :participant_declarations, -> { paid }, foreign_key: :statement_id
```

Comment:
do we really want to attach a scope a base association? getting any other kind of data from the association becomes difficult. you yourself have had to write
```
ParticipantDeclaration::NPQ.where(statement: @statement).voided
```
when you could have written
```
statement.participant_declarations.voided
```